packages/core/src/lib/markdown-it/highlight/HighlightRule.ts

```diff
@@ -61,56 +66,37 @@ export class HighlightRule {
     return atLineNumber;
   }
 
-  applyHighlight(line: string, lineNumber: number) {
+  getHighlightType(lineNumber: number): {
```

Comment:
Would be great if we could add some unit tests for this method as noted by Codecov check warnings

Reply:
Done

Reply:
Left with one case:
![image](https://github.com/MarkBind/markbind/assets/41845017/232014b9-2473-4ffc-9b4e-d470c990aeb5)

Reply:
I think done?